Use split ctest -S driver to allow testing on HPC machines

bartlettroscoe opened the pull request:
@fryeguy52 

## Description

These commits change the ATDM Trilinos driver scripts on serrano to use a new split ctest -S driver that builds on the login/compile node but runs tests on the compute node.  This should submit to the same CDash build but it does not seem to be working when run from serrano, even with CMake 3.10.

But the split driver submits to the same CDash build just fine when run the CEE machine ceerws1113.

This pulls in the updated TRIBITS_CTEST_DRIVER() function from TriBITSPub/TriBITS#259.

## How Has This Been Tested?

On crf450 I did a split driver submit with:

```
$ time env     JOB_NAME=gnu-debug-serial     WORKSPACE=$PWD \
      Trilinos_PACKAGES=Kokkos,Teuchos     CTEST_TEST_TYPE=Experimental \
      CTEST_DO_SUBMIT=ON     CTEST_DO_UPDATES=OFF \
     CTEST_START_WITH_EMPTY_BINARY_DIRECTORY=FALSE  \
  ~/Trilinos.base/Trilinos/cmake/ctest/drivers/atdm/rhel6/split-local-driver.sh  &> console.out

real    2m32.961s
user    4m21.216s
sys     1m0.138s
```

That submitted two builds:

* https://testing-vm.sandia.gov/cdash/index.php?project=Trilinos&parentid=3481929
* https://testing-vm.sandia.gov/cdash/index.php?project=Trilinos&parentid=3481932

On crf450, I did a single driver submit with:

```
$  time env     JOB_NAME=Trilinos-atdm-rhel6-gnu-debug-serial     WORKSPACE=$PWD \
     Trilinos_PACKAGES=Kokkos,Teuchos     CTEST_TEST_TYPE=Experimental     CTEST_DO_SUBMIT=ON \
     CTEST_DO_UPDATES=OFF     CTEST_START_WITH_EMPTY_BINARY_DIRECTORY=FALSE \
   ~/Trilinos.base/Trilinos/cmake/ctest/drivers/atdm/smart-jenkins-driver.sh  &> console.out

real    2m17.672s
user    3m23.318s
sys     0m23.729s
```

That submitted to:

* https://testing-vm.sandia.gov/cdash/index.php?project=Trilinos&parentid=3481939

On 'serrano', I did a submit with:

```
$ time env     JOB_NAME=Trilinos-atdm-toss3-intel-debug-openmp-panzer WORKSPACE=$PWD  \
    CTEST_TEST_TYPE=Experimental     CTEST_DO_SUBMIT=ON     CTEST_DO_UPDATES=OFF \
    CTEST_START_WITH_EMPTY_BINARY_DIRECTORY=FALSE \
  ~/Trilinos.base/Trilinos/cmake/ctest/drivers/atdm/smart-jenkins-driver.sh     &> console.out

real    19m23.286s
user    103m10.480s
sys     51m1.336s
```

that submitted to:

* https://testing-vm.sandia.gov/cdash/index.php?project=Trilinos&parentid=3481937
* https://testing-vm.sandia.gov/cdash/index.php?project=Trilinos&parentid=3481953

Note that on crf450, I verified that if you use CMake 3.11.1, it submits to the same CDash build.  An example off that from a few days ago is:

* https://testing-vm.sandia.gov/cdash/index.php?project=Trilinos&parentid=3478791

## Checklist

- [ ] My commit messages mention the appropriate GitHub issue numbers.
- [ ] My code follows the code style of the affected package(s).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.

trilinos-autotester commented:
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection Is Not Necessary for this Pull Request.

trilinos-autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.9.3</h3>
<ul>
  <li>Build Num: 595</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>COMPILER_MODULE</td>
    <td>sems-gcc/4.9.3</td>
  </tr>

  <tr>
    <td>JENKINS_BUILD_TYPE</td>
    <td>Release</td>
  </tr>

  <tr>
    <td>JENKINS_COMM_TYPE</td>
    <td>MPI</td>
  </tr>

  <tr>
    <td>JENKINS_DO_COMPLEX</td>
    <td>OFF</td>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>MPI_MODULE</td>
    <td>sems-openmpi/1.8.7</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2672</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>tril-204-dual-ctest-s-driver</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/bartlettroscoe/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>69ff3371e793dff72887b95d279c6946218e64af</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>07ae065bcefde62a07f0f8b8d147375d7ae5f39c</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.8.4</h3>
<ul>
  <li>Build Num: 312</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>COMPILER_MODULE</td>
    <td>sems-gcc/4.8.4</td>
  </tr>

  <tr>
    <td>JENKINS_BUILD_TYPE</td>
    <td>Release</td>
  </tr>

  <tr>
    <td>JENKINS_COMM_TYPE</td>
    <td>MPI</td>
  </tr>

  <tr>
    <td>JENKINS_DO_COMPLEX</td>
    <td>OFF</td>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>MPI_MODULE</td>
    <td>sems-openmpi/1.8.7</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2672</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>tril-204-dual-ctest-s-driver</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/bartlettroscoe/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>69ff3371e793dff72887b95d279c6946218e64af</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>07ae065bcefde62a07f0f8b8d147375d7ae5f39c</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: TRILINOS (bartlettroscoe/Trilinos)</dt>
  <ul>
    <li>Branch: tril-204-dual-ctest-s-driver</li>
    <li>SHA: 69ff3371e793dff72887b95d279c6946218e64af</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: bartlettroscoe
</details>

trilinos-autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.9.3</h3>
<ul>
  <li>Build Num: 598</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>COMPILER_MODULE</td>
    <td>sems-gcc/4.9.3</td>
  </tr>

  <tr>
    <td>JENKINS_BUILD_TYPE</td>
    <td>Release</td>
  </tr>

  <tr>
    <td>JENKINS_COMM_TYPE</td>
    <td>MPI</td>
  </tr>

  <tr>
    <td>JENKINS_DO_COMPLEX</td>
    <td>OFF</td>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>MPI_MODULE</td>
    <td>sems-openmpi/1.8.7</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2672</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>tril-204-dual-ctest-s-driver</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/bartlettroscoe/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>247160bfb61c4f1d4b5b2d0266519de7524e3d7c</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>07ae065bcefde62a07f0f8b8d147375d7ae5f39c</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.8.4</h3>
<ul>
  <li>Build Num: 315</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>COMPILER_MODULE</td>
    <td>sems-gcc/4.8.4</td>
  </tr>

  <tr>
    <td>JENKINS_BUILD_TYPE</td>
    <td>Release</td>
  </tr>

  <tr>
    <td>JENKINS_COMM_TYPE</td>
    <td>MPI</td>
  </tr>

  <tr>
    <td>JENKINS_DO_COMPLEX</td>
    <td>OFF</td>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>MPI_MODULE</td>
    <td>sems-openmpi/1.8.7</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2672</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>tril-204-dual-ctest-s-driver</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/bartlettroscoe/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>247160bfb61c4f1d4b5b2d0266519de7524e3d7c</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>07ae065bcefde62a07f0f8b8d147375d7ae5f39c</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: TRILINOS (bartlettroscoe/Trilinos)</dt>
  <ul>
    <li>Branch: tril-204-dual-ctest-s-driver</li>
    <li>SHA: 247160bfb61c4f1d4b5b2d0266519de7524e3d7c</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: bartlettroscoe
</details>

bartlettroscoe commented:
Kitware helped to diagnose the problem with not submitting to the same CDash build and I added the TriBITS commit https://github.com/TriBITSPub/TriBITS/commit/8d4f671ae89549f58c38522582933f78fb9f78e9 to fix that.  Therefore, I created a new snapshot commit and rebased the other commits on top of that and forced pushed this topic branch.  With that, the submit on 'serrano' showed:

```
***
*** Use previous dashboard calling ctest_start(... APPEND) due to CTEST_DO_NEW_START='OFF' ...
***


Previous file:

  '/ascldap/users/rabartl/Trilinos.base/BUILDS/SERRANO/JENKINS_DRIVER/SRC_AND_BUILD/BUILD/Testing/TAG'

exists with contents:

20180503-1836
Experimental



Calling ctest_start(Experimental;TRACK;Experimental;APPEND)... 


Run dashboard with model Experimental
   Source directory: /ascldap/users/rabartl/Trilinos.base/BUILDS/SERRANO/JENKINS_DRIVER/SRC_AND_BUILD/Trilinos
   Build directory: /ascldap/users/rabartl/Trilinos.base/BUILDS/SERRANO/JENKINS_DRIVER/SRC_AND_BUILD/BUILD
   Track: Experimental
   Reading ctest configuration file: /ascldap/users/rabartl/Trilinos.base/BUILDS/SERRANO/JENKINS_DRIVER/SRC_AND_BUILD/Trilinos/CTestConfig.cmake
-- CTEST_DROP_SITE='testing.sandia.gov'
-- CTEST_PROJECT_NAME='Trilinos'
-- CTEST_DROP_LOCATION='/cdash/submit.php?project=Trilinos'
-- CTEST_TRIGGER_SITE=''
-- CTEST_DROP_SITE_CDASH='TRUE'
-- TRIBITS_2ND_CTEST_DROP_SITE='testing-vm.sandia.gov'
-- TRIBITS_2ND_CTEST_DROP_LOCATION='/cdash/submit.php?project=Trilinos'
   Site: serrano
   Build name: Trilinos-atdm-toss3-intel-debug-openmp
  Use existing tag: 20180503-1836 - Experimental
   Use Experimental tag: 20180503-1836
```

and that submitted to the same CDash build:

* https://testing-vm.sandia.gov/cdash/index.php?project=Trilinos&parentid=3482480&filtercount=1&showfilters=1&field1=groupname&compare1=61&value1=Experimental

That shows test failures but we will address that after this is running in Jenkins builds.

@fryeguy52, can you please approve this PR?

trilinos-autotester commented:
Status Flag 'Pull Request AutoTester' - ERROR: A user has commited a change to the PR before testing completed.  The testing SHA = 247160bfb61c4f1d4b5b2d0266519de7524e3d7c Does not match the current commit SHA = 7f9d6908704718537fb81b8d4e6d527d4e822979.  Testing of this PR must occur again.

trilinos-autotester commented:
Status Flag 'Pre-Test Inspection' - Auto Inspected - Inspection Is Not Necessary for this Pull Request.

trilinos-autotester commented:
Status Flag 'Pull Request AutoTester' - Testing Jenkins Projects:

<details>
<summary>
  <b>Pull Request Auto Testing STARTING</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.9.3</h3>
<ul>
  <li>Build Num: 602</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>COMPILER_MODULE</td>
    <td>sems-gcc/4.9.3</td>
  </tr>

  <tr>
    <td>JENKINS_BUILD_TYPE</td>
    <td>Release</td>
  </tr>

  <tr>
    <td>JENKINS_COMM_TYPE</td>
    <td>MPI</td>
  </tr>

  <tr>
    <td>JENKINS_DO_COMPLEX</td>
    <td>OFF</td>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>MPI_MODULE</td>
    <td>sems-openmpi/1.8.7</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2672</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>tril-204-dual-ctest-s-driver</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/bartlettroscoe/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>7f9d6908704718537fb81b8d4e6d527d4e822979</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>07ae065bcefde62a07f0f8b8d147375d7ae5f39c</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.8.4</h3>
<ul>
  <li>Build Num: 319</li>
  <li>Status: STARTED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>COMPILER_MODULE</td>
    <td>sems-gcc/4.8.4</td>
  </tr>

  <tr>
    <td>JENKINS_BUILD_TYPE</td>
    <td>Release</td>
  </tr>

  <tr>
    <td>JENKINS_COMM_TYPE</td>
    <td>MPI</td>
  </tr>

  <tr>
    <td>JENKINS_DO_COMPLEX</td>
    <td>OFF</td>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>MPI_MODULE</td>
    <td>sems-openmpi/1.8.7</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2672</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>tril-204-dual-ctest-s-driver</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/bartlettroscoe/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>7f9d6908704718537fb81b8d4e6d527d4e822979</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>07ae065bcefde62a07f0f8b8d147375d7ae5f39c</td>
  </tr>
</table>

<h2>Using Repos:</h2>

<dl>
  <dt>Repo: TRILINOS (bartlettroscoe/Trilinos)</dt>
  <ul>
    <li>Branch: tril-204-dual-ctest-s-driver</li>
    <li>SHA: 7f9d6908704718537fb81b8d4e6d527d4e822979</li>
    <li>Mode: TEST_REPO</li>
  </ul>
</dl>

Pull Request Author: bartlettroscoe
</details>

trilinos-autotester commented:
Status Flag 'Pull Request AutoTester' - Jenkins Testing: all Jobs PASSED

<details>
<summary>
  <b>Pull Request Auto Testing has PASSED</b> (click to expand)
</summary>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.9.3</h3>
<ul>
  <li>Build Num: 602</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>COMPILER_MODULE</td>
    <td>sems-gcc/4.9.3</td>
  </tr>

  <tr>
    <td>JENKINS_BUILD_TYPE</td>
    <td>Release</td>
  </tr>

  <tr>
    <td>JENKINS_COMM_TYPE</td>
    <td>MPI</td>
  </tr>

  <tr>
    <td>JENKINS_DO_COMPLEX</td>
    <td>OFF</td>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>MPI_MODULE</td>
    <td>sems-openmpi/1.8.7</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2672</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>tril-204-dual-ctest-s-driver</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/bartlettroscoe/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>7f9d6908704718537fb81b8d4e6d527d4e822979</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>07ae065bcefde62a07f0f8b8d147375d7ae5f39c</td>
  </tr>
</table>

<h2>Build Information</h2>
<h3>Test Name: Trilinos_pullrequest_gcc_4.8.4</h3>
<ul>
  <li>Build Num: 319</li>
  <li>Status: PASSED</li>
</ul>

<h4>Jenkins Parameters</h4>
<table>
  <tr>
    <th>Parameter Name</th>
    <th>Value</th>
  </tr>

  <tr>
    <td>COMPILER_MODULE</td>
    <td>sems-gcc/4.8.4</td>
  </tr>

  <tr>
    <td>JENKINS_BUILD_TYPE</td>
    <td>Release</td>
  </tr>

  <tr>
    <td>JENKINS_COMM_TYPE</td>
    <td>MPI</td>
  </tr>

  <tr>
    <td>JENKINS_DO_COMPLEX</td>
    <td>OFF</td>
  </tr>

  <tr>
    <td>JENKINS_JOB_TYPE</td>
    <td>Experimental</td>
  </tr>

  <tr>
    <td>MPI_MODULE</td>
    <td>sems-openmpi/1.8.7</td>
  </tr>

  <tr>
    <td>PULLREQUESTNUM</td>
    <td>2672</td>
  </tr>

  <tr>
    <td>TEST_REPO_ALIAS</td>
    <td>TRILINOS</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_BRANCH</td>
    <td>tril-204-dual-ctest-s-driver</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_REPO</td>
    <td>https://github.com/bartlettroscoe/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_SOURCE_SHA</td>
    <td>7f9d6908704718537fb81b8d4e6d527d4e822979</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_BRANCH</td>
    <td>develop</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_REPO</td>
    <td>https://github.com/trilinos/Trilinos</td>
  </tr>

  <tr>
    <td>TRILINOS_TARGET_SHA</td>
    <td>07ae065bcefde62a07f0f8b8d147375d7ae5f39c</td>
  </tr>
</table>
</details>


<br>[CDash Test Results for PR# 2672.](https://testing-vm.sandia.gov/cdash/index.php?project=Trilinos&filtercombine=and&filtercombine=and&filtercombine=and&filtercombine=and&filtercombine=and&filtercombine=and&filtercount=2&showfilters=1&filtercombine=and&field1=buildname&compare1=63&value1=PR-2672-test&field2=buildstarttime&compare2=84&value2=NOW
)<br>

trilinos-autotester commented:
Status Flag 'Pre-Merge Inspection' - SUCCESS: The last commit to this Pull Request has been INSPECTED AND APPROVED by [ fryeguy52 ]!

trilinos-autotester commented:
Status Flag 'Pull Request AutoTester' - Pull Request MUST BE MERGED MANUALLY BY Project Team - Master Automerge is disabled (in .cfg file)
